src/signer.rs

```diff
@@ -99,12 +99,12 @@ mod tests {
             0xb, 0x8, 0x80, 0x92, 0xb8, 0xc3, 0x98, 0xfe, 0xff, 0xff, 0xff, 0x1,
         ];
 
-        signer.sign(&some_message1).unwrap();
```

Comment:
Note per above the fallible version of `sign` was changed to `try_sign`, and `sign` now auto-unwraps